### DISCUSSION TITLE
Modify only the POST route the for login action?

### DISCUSSION THREAD

**Discussion by NickLikesPHP** [codeigniter4/shield#1114] [Q&A] [answered]
When you use `service('auth')->routes($routes, ['except' => ['login']]);` it removes both GET and POST routes. Is it possible to just exclude the POST route, or will I have to re-add both the GET and POST routes? Thanks!

```
$routes->get('login', '\CodeIgniter\Shield\Controllers\LoginController::loginView');
$routes->post('login', 'Shield\LoginController::loginAction');
```

**Comment by kenjis** [comment, marked as answer]
> Is it possible to just exclude the POST route

No.
